tremor-cli/src/server.rs

```diff
@@ -121,19 +121,28 @@ pub(crate) async fn run_dun(matches: &ArgMatches) -> Result<()> {
     let (world, handle) = World::start(64, storage_directory).await?;
 
     if let Some(config_files) = matches.values_of("artefacts") {
+        let mut yaml_files = Vec::with_capacity(16);
         // We process trickle files first
-        for config_file in config_files.clone() {
-            match get_source_kind(config_file) {
-                SourceKind::Trickle => tremor_runtime::load_query_file(&world, config_file).await?,
-                _ => continue,
+        for config_file in config_files {
+            let kind = get_source_kind(config_file);
+            match kind {
+                SourceKind::Trickle => {
+                    if let Err(e) = tremor_runtime::load_query_file(&world, config_file).await {
+                        error!("Failed to load trickle script file {}: {}", config_file, e);
```

Comment:
We wanna `return Err(...)` here?

Reply:
Specifically not, no, returning an error here is what leads to panicking the runtime.